helm/fiftyone-teams-app/README.md

```diff
@@ -124,6 +125,32 @@ When performing an upgrade, please review
 
 Consider if you will require these settings for your deployment.
 
+### Builtin Delegated Operator Orchestrator
+
+FiftyOne Teams v2.2 introduces a builtin orchestrator to run
+[Delegated Operations](https://docs.voxel51.com/teams/teams_plugins.html#delegated-operations),
+instead of (or in addition to) configuring your own orchestrator such as Airflow.
+
+This option can be added to any of the 3 existing
+[plugin modes](fiftyone-teams-plugins). If you're using the builtin-operator
+only option, the Persistent Volume Claim should be omitted.
+
+To enable this mode
+
+- In `values.yaml`, set
+  - `delegatedOperatorExecutorSettings.enabled: true`
+  - The path for a Persistent Volume Claim mounted to the
+    `teams-do` deployment in
+    - `delegatedOperatorExecutorSettings.env.FIFTYONE_PLUGINS_DIR`
+- See
+  [Adding Shared Storage for FiftyOne Teams Plugins](../docs/plugins-storage.md)
+  - Mount a Persistent Volume Claim (PVC) that provides
+    - `ReadWrite` permissions to the `teams-do` deployment
+      at the `FIFTYONE_PLUGINS_DIR` path
+
+To use plugins with custom dependencies, build and use
+[Custom Plugins Images](https://github.com/voxel51/fiftyone-teams-app-deploy/blob/main/docs/custom-plugins.md).
+
```

Comment:
I feel like they ahve to set a secret in here somewhere too?

Reply:
I removed that requirement here [381fd90](https://github.com/voxel51/fiftyone-teams-app-deploy/pull/210/commits/381fd904ea95240f6412e0a96c3211c0ae270856)

I implemented the internal scoped API key concept we discussed previously. No need for admin API key anymore.

Reply:
formerly known as "shadow key" 😱 😮